.eslintrc.ts.json

```diff
@@ -113,6 +113,7 @@
         "assets/js/src/common/form/yesno/yesno.tsx",
         "assets/js/src/common/functions/change_handlers.ts",
         "assets/js/src/common/grid/_stories/grid.tsx",
+        "assets/js/src/common/grid/centered_row.tsx",
```

Comment:
We shouldn't add more ignored files to this list. You created a new component that shouldn't be added to this long list.